### PR TITLE
Improve AdminExtension doc

### PR DESCRIPTION
**Pull request by eerison**
## Improving AdminExtension docs

I am targeting this branch, because it's the current stable version of sonata admin.

Issue: https://github.com/sonata-project/SonataPageBundle/issues/1639

No changelog for doc

### CODE REVIEW

**Comment by eerison** [comment]
I didn't finish yet, But I would like to know if it looks what you expect from the "improve" of admin extension docs!

@VincentLanglet

**Comment by VincentLanglet** [comment]
> I didn't finish yet, But I would like to know if it looks what you expect from the "improve" of admin extension docs!
> 
> @VincentLanglet

Yes :)

**Comment by eerison** [comment]
It's for you better visualize how it looks like

ScreenShot Tool -20221108140025

File: docs/reference/extensions.rst [hunk 1]
```diff
@@ -1,137 +1,134 @@
-Extensions
```
Comment (eerison) [inline docs/reference/extensions.rst:1]: Hey @VincentLanglet could I rename this file to `customize_admin.rst`?

File: docs/reference/extensions.rst [hunk 2]
```diff
@@ -152,3 +149,34 @@ the methods `addExtension` and `removeExtension`::
...
+                ->add('position')
+            ->end()
+        ;
+    }
+
+In this case the position field will be into the **main** group, after **url** field
+
+.. image:: ../images/customize_admin__reorder_positon_field.png
+
+.. _Groups and Tabs: https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/getting_started/the_form_view/?highlight=Tabs#using-groups-and-tabs
```
Comment (eerison) [inline docs/reference/extensions.rst]: How Can I link to other file? just using full url?

Comment (VincentLanglet) [reply docs/reference/extensions.rst]: I don't really know since I didn't add a lot of doc. Isn't there example in other docs ?

**Review by jordisala1991 — DISMISSED** [review]

File: docs/reference/extensions.rst (hunk 1 above)
Comment (eerison) [reply docs/reference/extensions.rst:1]: it's missing only this question be answered 👀

Comment (eerison) [reply docs/reference/extensions.rst:1]: we don't have too much benefits with it, it's more to keep the title equals `file name`!

But if I do this and others docs links with this page, it'll return not found!

**Review by VincentLanglet — APPROVED** [review]
Can I merge it @eerison ?

**Comment by eerison** [comment]
Yep!

**Pull request merged**